src/layout/sidebar/components/SidebarMenuButton.tsx

```diff
@@ -0,0 +1,42 @@
+import { FC } from "react";
+import styles from "./SidebarMenuButton.module.css";
+
+interface Props {
+  icon: Element | any;
+  title: string;
+  counter: number;
+  isClicked: State;
+}
+
+type State = "normal" | "active";
+
+const SidebarMenuButton: FC<Props> = (props) => {
+  //No logic has been set here, this is just for preview purposes
+  let state: State = props.isClicked;
+
+  return (
+    <div
```

Comment:
This can be a `button` or a `a` tag because it should work as a button. This is not a mandatory thing but it would be more accurate for a11y.